app/scrapers/__init__.py

```diff
@@ -42,6 +42,8 @@ def feed_gen(query, engine, count=10, qtype=''):
     engine = old_names.get(engine, engine)
     if engine in ('quora', 'youtube'):
         urls = scrapers[engine].search_without_count(query)
+    elif engine in ('ask',) and qtype == 'vid':
```

Comment:
Why use `in ('ask',)`
You can go with `if engine == 'ask'`
since there is only one option to choose from?

Reply:
So that other engines can also be added in this tuple, if there implementation is same.